sequential/bank_accounts/src/bank_account.erl

```diff
@@ -2,5 +2,16 @@
 
 -export([process_operation/2]).
 
-process_operation(Bank, Operation) ->
-  put_your_solution_here.
+process_operation(Bank,
+		  {AccNumber, withdraw, Amount}) ->
+    case lists:keyfind(AccNumber, 1, Bank) of
+      false -> {error, account_not_found};
```

Comment:
when doing case clauses, it's better for readability to put the implementations either all on the next line or all inline (if they're short enough), but not mix them